src/functions-and-arrays.js

```diff
@@ -1,73 +1,450 @@
 // Iteration #1: Find the maximum
-
+const maxOfTwoNumbers = (num1, num2) => {
+  if (num1 > num2) return num1;
+  else if (num2 > num1) return num2;
+  else return num1;
+};
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
+];
+
+const findLongestWord = words => {
+  let longestWord = "";
+  if (words.length === 0) return null;
```

Comment:
Since you are returning:
return null

You wont need the else statement since if the condition is true you return out of the function anyways